### DISCUSSION TITLE
Paramiko Tutorial Discussion

### DISCUSSION THREAD

**Discussion by toddbirchard** [hackersandslackers/paramiko-tutorial#36] [General]
Having trouble 😩? Found a bug 🐛? Want to say hi 👋? Whether you're here for words of encouragement or hurtful hate speech, feel free to jump into the chaos that is the [Paramiko Tutorial](https://hackersandslackers.com/automate-ssh-scp-python-paramiko/) discussion.

**Comment by lgxboxslim** [comment]
man, i try understand paramiko, but i cant broo, need help

**Comment by DataBach-maker** [comment]
Hello @toddbirchard , thanks a lot for the tutorial. I really appreciate it ! Question, if it is not too much too ask, how can I adabt your solution to do a proxy hob ? Or tunnel forwarding ?
